.github/workflows/ci.yaml

```diff
@@ -0,0 +1,15 @@
+name: Lint, build, and test
```

Comment:
this checks things and nags you if a commit breaks everything (email and/or PR check and/or as a check/x on the commit)